rafthttp/stream_test.go

```diff
@@ -59,11 +59,9 @@ func TestStreamWriterAttachOutgoingConn(t *testing.T) {
 			}
 		}
 
-		// msgc has been swapped with a new one now that prevwfc is closed
-		msgc, ok := sw.writec()
-		if !ok {
-			t.Errorf("#%d: working status = %v, want true", i, ok)
-		}
+		// msgc has either been swapped with a new one after closing prevwfc or
+		// is the initial msgc (in which case the first connection may be pending)
+		msgc, _ := sw.writec()
```

Comment:
OK. I looked at the code and you are right. 

But I confused with the comment
```
// msgc has either been swapped with a new one after closing prevwfc or
// is the initial msgc (in which case the first connection may be pending)
```

I do not quite understand why we need the `or ...`. msgc should always be the new one after prevwfc gets closed. right?